src/fields/abstractField.js

```diff
@@ -1,5 +1,15 @@
 import { get as objGet, each, isFunction, isString, isArray } from "lodash";
 
+function checkIfValidatorIsAStringAndConvertToObject(validator) {
+	let transformedValidator = validator;
+	if(isString(transformedValidator)){
+		let chunks = transformedValidator.split(".");
+		let root = chunks.shift();
+		transformedValidator = objGet(window[root], chunks.join("."));
```

Comment:
Maybe add some warning here that such a validator cannot be found and some tip on what to do.
Don't know how the default error looks for this case but I guess it's "ReferenceError x is not defined" which is not very helpful I guess.